### ISSUE TITLE
ptags doesn't compile, fails with "error[E0277]: the trait bound `Opt: structopt_toml::structopt::StructOpt` is not satisfied"

### ISSUE THREAD

**Issue by nad2000** [dalance/ptags#38]
```bash
> cargo --version
cargo 1.51.0 (43b129a20 2021-03-16)

> cargo install ptags

    Updating crates.io index
  Installing ptags v0.3.2
   Compiling libc v0.2.92
   Compiling proc-macro2 v1.0.24
...

   Compiling structopt-toml v0.4.5
   Compiling ptags v0.3.2
error[E0277]: the trait bound `Opt: structopt_toml::structopt::StructOpt` is not satisfied
  --> /home/rcir178/.cargo/registry/src/github.com-1ecc6299db9ec823/ptags-0.3.2/src/bin.rs:21:52
   |
21 | #[derive(Debug, Deserialize, Serialize, StructOpt, StructOptToml)]
   |                                                    ^^^^^^^^^^^^^ the trait `structopt_toml::structopt::StructOpt` is not implemented for `Opt`
   |
   = help: see issue #48214
   = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error

For more information about this error, try `rustc --explain E0277`.
error: could not compile `ptags`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `ptags v0.3.2`, intermediate artifacts can be found at `/tmp/cargo-installANJhYz`

Caused by:
  build failed
```

**Comment by davidscotson** [comment]
I hit this as well (though running a slightly older version of nightly), thought it worth mentioning for anyone else who finds this issue via the error message, that switching to a stable (non-nightly) version of rust let me install it without errors.

**Comment by shivarajagopal** [comment]
Running into the same issue here. @davidscotson - what version worked for you?
```
> cargo --version
cargo 1.51.0 (43b129a20 2021-03-16)

> rustc--version
rustc 1.51.0 (2fd73fabe 2021-03-23)
```

**Comment by dalance** [comment]
Thanks!
Some dependencies seem to be broken.
I fixed it, and released v0.3.3.
Could you try it?

**Comment by shivarajagopal** [comment]
Looks good for me now! Thanks @dalance!
